src/pytest_respect/resources.py

```diff
@@ -542,7 +493,7 @@ def save_text(
     ) -> None:
         """Write a string to a resource relative to the current test."""
         path = self.path(*parts, ext=ext, path_maker=path_maker)
-        if not path.parent.exists:
+        if not path.parent.exists():
```

Comment:
Obvious bug, found when I wanted to increase the test coverage for purely nerdy reasons.